packages/utilities/src/lib/objectValues.ts

```diff
@@ -0,0 +1,5 @@
+import type { NonNullObject } from './utilityTypes';
+
+export function objectValues<T extends NonNullObject>(obj: T): T extends ArrayLike<any> ? T[number][] : T[keyof T][] {
```

Comment:
Shouldn't this use `T extends ArrayLike<infer Values> ? Values[] : T[keyof T][]`?

Reply:
I guess both work

Reply:
What is `Values` in this context?